tests/bulk/write-0001.phpt

```diff
@@ -116,7 +116,7 @@ object(MongoDB\Driver\BulkWrite)#%d (%d) {
   ["executed"]=>
   bool(true)
   ["server_id"]=>
-  int(1)
+  int(%r[1-9]\d*%r)
```

Comment:
Not specifically for 4.0.0-rc5, but when the second node in my replica set was the primary this reported 2 instead of 1. We really just want to assert that it's non-zero.